contrib/reprepro/suites.sh

```diff
@@ -5,7 +5,7 @@ cd "$(dirname "$BASH_SOURCE")/../.."
 
 targets_from() {
        git fetch -q https://github.com/docker/docker.git "$1"
-       git ls-tree -r --name-only "$(git rev-parse FETCH_HEAD)" contrib/builder/deb | grep '/Dockerfile$' | sed -r 's!^contrib/builder/deb/|-debootstrap|/Dockerfile$!!g'
+       git ls-tree -r --name-only "$(git rev-parse FETCH_HEAD)" contrib/builder/deb/${DOCKER_ENGINE_GOARCH:="amd64"} | grep '/Dockerfile$' | sed -r 's!^contrib/builder/deb/|-debootstrap|/Dockerfile$!!g'
```

Comment:
The `sed` worked correctly building this PR. Could it be that it worked because we are in a branch and FETCH_HEAD commit id still has the old structure?

The build-deb.sh still only generates the amd64 Dockerfiles. The arm/debian-jessie Dockerfile is built manually at the moment. When we introduce more than one ARM OS platform we will update the build-deb.sh in another PR. That was our plan to get at least the new folder structure in it. Rebasing moving folders is no fun. ;-)